cogs/annual_handover_and_reset.py

```diff
@@ -115,19 +114,19 @@ async def committee_handover(self, ctx: TeXBotApplicationContext) -> None:
             logger.debug("Removing Committee role from user: %s", committee_member)
             await committee_member.remove_roles(
                 committee_role,
-                reason=f"{ctx.user} used TeX-Bot slash-command: \"committee-handover\"",
+                reason=f'{ctx.user} used TeX-Bot slash-command: "committee-handover"',
             )
 
             if automod_role and automod_role in committee_member.roles:
                 await committee_member.remove_roles(
                     automod_role,
-                    reason=f"{ctx.user} used TeX-Bot slash-command: \"committee-handover\"",
+                    reason=f'{ctx.user} used TeX-Bot slash-command: "committee-handover"',
                 )
 
         await initial_response.edit(
             content=(
-                ":hourglass: Giving committee-elect users the \"Committee\" role "
-                "and removing their \"Committee-Elect\" role... :hourglass:"
+                ':hourglass: Giving committee-elect users the "Committee" role '
+                'and removing their "Committee-Elect" role... :hourglass:'
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes